pkg/peertls/peertls_test/peertls_test.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2018 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package peertls
+package peertls_test
```

Comment:
The extra folder isn't necessary.

Reply:
You can't have multiple packages in the same folder,  what are you suggesting?

Reply:
Cool, didn't know that